### PR TITLE
feat: adding astro files support via JSX language syntax

### LINKED ISSUE #336: It is possible to add custom extensions to syntax plugins?
For example `.astro` files works great with the JSX language syntax. It is possible via config (init.lua) to include `.astro` files on this language_jsx.lua ?

```
syntax.add {
  name = "JSX",
  files = { "%.jsx$" },
```
to
```
files = { "%.jsx$", "%.astro$" },
```

Or i'm forced to duplicate the language file and include a modified version?

### PR DESCRIPTION
**Pull request by felixsanz**
Fixes #336

### CODE REVIEW

**Comment by Guldoman** [comment]
Would you mind bumping the plugin version in the manifest too?

File: plugins/language_jsx.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@ local syntax = require "core.syntax"
 
 syntax.add {
   name = "JSX",
-  files = { "%.jsx$" },
+  files = { "%.jsx$", "%.astro$" },
   comment = "//",
   patterns = {
     { pattern = "//.-\n",               type = "comment"  },
```
Comment (Guldoman) [inline plugins/language_jsx.lua:10]: I realize that this isn't really part of the PR, but could you also change this?
```suggestion
    { pattern = "//.*",                 type = "comment"  },
```

File: plugins/language_jsx.lua [hunk 2]
```diff
@@ -4,7 +4,7 @@ local syntax = require "core.syntax"
 
 syntax.add {
   name = "JSX",
-  files = { "%.jsx$" },
+  files = { "%.jsx$", "%.astro$" },
   comment = "//",
```
Comment (Guldoman) [inline plugins/language_jsx.lua:8]: Oh and this too:
```suggestion
  comment = "//",
  block_comment = { "/*", "*/" },
```

**Comment by felixsanz** [comment]
Done. Did I updated the manifest correctly? I bumped two versions

**Comment by Guldoman** [comment]
> Did I updated the manifest correctly? I bumped two versions

In theory bumping the `meta` plugin version shouldn't have been necessary, but I forgot to bump it in a previous PR, so all good.

File: plugins/language_jsx.lua [hunk 3]
```diff
@@ -4,10 +4,12 @@ local syntax = require "core.syntax"
...
+  block_comment = { "/*", "*/" },
+  block_comment = { "/*", "*/" },
```
Comment (Guldoman) [inline plugins/language_jsx.lua:10]: Got added twice.

**Comment by felixsanz** [comment]
Done. Sorry I was using the web UI for the simple edit and I messed it up xD

**Comment by Guldoman** [comment]
No worries! Thank you!

**Pull request merged**